package.json

```diff
@@ -22,21 +22,21 @@
     "turbo-link": "turbo login && turbo link",
     "verify": "turbo run verify",
     "WEB========================================": "",
-    "web:dev": "turbo run start:dev --filter=audius-client",
-    "web:prod": "turbo run start:prod --filter=audius-client",
-    "web:stage": "turbo run start:stage --filter=audius-client",
+    "web:dev": "npm run start:dev -w audius-client",
```

Comment:
how come these no turbo? they will pick up underlying deps from turbo automatically now?

Reply:
This removes the need to define each of these individually as pipelines in `turbo.json`. In the client package.json, `start:dev` for example calls `turbo run start` with the correct env vars and arguments. This means we only need a single pipeline defined for `start` rather than one for each environment